client/sandbox/react-nextjs/pages/play/missing-attrs.tsx

```diff
@@ -0,0 +1,197 @@
+import { useEffect, useState } from "react";
+import config from "../../config";
+import { init, init_experimental, tx, id, i } from "@instantdb/react";
+import { useRouter } from "next/router";
+
+const schema = i.graph(
+  {
+    comments: i.entity({
+      slug: i.string().unique().indexed(),
+    }),
+    $users: i.entity({
+      email: i.string().unique().indexed(),
+    }),
+  },
+  {
+    commentAuthors: {
+      forward: {
+        on: "comments",
+        has: "one",
+        label: "author",
+      },
+      reverse: {
+        on: "$users",
+        has: "many",
+        label: "authoredComments",
+      },
+    },
+  },
+);
+
+function Example({ appId, useSchema }: { appId: string; useSchema: boolean }) {
+  const myConfig = { ...config, appId };
+  const db = useSchema
+    ? init_experimental({ ...myConfig, schema })
+    : (init(myConfig) as any);
+  const q = db.useQuery({ comments: {} });
+  const [attrs, setAttrs] = useState<any>();
+  useEffect(() => {
+    const unsub = db._core._reactor.subscribeAttrs((res: any) => {
+      setAttrs(res);
+    });
+    return unsub;
+  });
+
+  return (
+    <div>
+      <div>
+        <button
+          className="bg-black text-white m-2 p-2"
+          onClick={() =>
```

Comment:
Could you add a button that tries to add an attribute that doesn't exist. Curious to see what the error looks like in the sandbox.

Reply:
<img width="654" alt="CleanShot 2024-11-19 at 15 00 03@2x" src="https://github.com/user-attachments/assets/a2c1110f-9bff-476d-b532-15db2632dd5b">

Adding! Looks like we have a try catch